opensrp-chw-malaria/src/main/java/org/smartregister/chw/malaria/provider/MalariaRegisterProvider.java

```diff
@@ -81,6 +81,11 @@ private void populatePatientColumn(CommonPersonObjectClient pc, final RegisterVi
         viewHolder.patientColumn.setTag(pc);
         viewHolder.patientColumn.setTag(R.id.VIEW_ID, BaseMalariaRegisterFragment.CLICK_VIEW_NORMAL);
 
+        if (age >= 10) {
```

Comment:
is the due status calculated as per age?

Reply:
No it isnt